Sage/carbon
# fix(inline-input): fix to input width that affects input icon - FE-2671

Fixes FE-2671
Fixes GitHub Issue: #2748

### Proposed behaviour
InlineInputs should display a validation icon correctly by default aligned to the right hand side of the input field.

![inline-input-fixed](https://user-images.githubusercontent.com/47245766/79457193-b7708900-7fe7-11ea-8d70-8a262b44641e.gif)


### Current behaviour
InlineInputs are using up 100% of the width pushing a validation icon to the new line. (wrap)

![Screenshot 2020-04-16 at 13 40 02](https://user-images.githubusercontent.com/47245766/79457228-cb1bef80-7fe7-11ea-8cbf-107ee075e29d.png)


### Checklist

<del>- [ ] Cypress automation tests
<del>- [ ] Typescript `d.ts` file added or updated

### Testing instructions
1. checkout branch
2. navigate to http://localhost:9001/?path=/story/inlineinputs--validations
3. Investigate first inline input that should have a icon to the right hand side

Inspecting element in the browser and changing input width: auto; to width: 100% will cause the issue to re-appear confirming that width: auto; resoles the issue.
